### PR TITLE
Backport of GLASSFISH-17606

### PR DESCRIPTION
**Pull request by iweiss**

### CODE REVIEW

File: src/main/java/javax/el/BeanELResolver.java [hunk 1]
```diff
@@ -55,21 +55,18 @@
 
 package javax.el;
 
+import java.beans.*;
```
Comment (ctomc) [inline src/main/java/javax/el/BeanELResolver.java]: please don't use * import. unless there are more than 20 classes from same package (we have IDE set to that)

Comment (iweiss) [inline src/main/java/javax/el/BeanELResolver.java]: Sorry. Should I resubmit?

Comment (ctomc) [inline src/main/java/javax/el/BeanELResolver.java]: just amend commit and force push so it updates this PR.

Comment (iweiss) [inline src/main/java/javax/el/BeanELResolver.java]: Done.

**Pull request merged**